firefox-ios/Client/TabManagement/Tab.swift

```diff
@@ -415,16 +419,12 @@ class Tab: NSObject, ThemeApplicable {
         if UserDefaults.standard.bool(forKey: PrefsKeys.FasterInactiveTabsOverride) {
             inactiveDate = Calendar.current.date(byAdding: .second, value: -10, to: currentDate) ?? Date()
         } else {
+            // FIXME Is there a reason we use noon of the current day instead of the exact time, when calculating -14 days?
             inactiveDate = Calendar.current.date(byAdding: .day, value: -14, to: currentDate.noon) ?? Date()
```

Comment:
Just wanted to call this out. Is there a reason for this extra logic to check based on `noon` of the current day, vs. the actual time? 🤔  After 14 days, what's a couple hours difference...?

Reply:
Maybe just for consistency for debugging (in cases across timezones or some other edge case?) someone thought it would be better? 
I also imagine it would be easier to look at a bunch of lines of data of multiple inactive tabs if you only had to account for the date and not the time? Really just guessing though, Orla might know the actual answer to this 😆 
Ex: 
12/2/2024 12:00:00, 
13/2/2024 12:00:00

vs 
12/2/2024 01:22:33, 
13/2/2024 23:02:13

Reply:
I have no idea what the original thinking was here 🙈

Reply:
Huh, interesting! 😂  I guess my vote is on removing it since we don't seem to have a good reason to be this specific on the time, but I can leave it if we're not confident of the potential impact.

Reply:
I'd say if this info isn't feeding into any telemetry probably safe to change it, if it is feeding into telemetry then keep it. 👍 I don't remember it feeding into anything when I looked yesterday.

Reply:
Thanks Sophie, I didn't think of that possibility. 🤔  I'll make a [spike task](https://mozilla-hub.atlassian.net/browse/FXIOS-10005) to look into this more later since it's just a nit thing.